### DISCUSSION TITLE
RejectedExecutionException error

### DISCUSSION THREAD

**Discussion by OMAR-3OOV** [discord-jda/JDA#2155] [Questions and Help] [answered]
Hey.

I am just trying to use the shutdown method, meanwhile, I face a **RejectedExecutionException**  breakpoint, with the message "_The Requester has been stopped! No new requests can be requested!_". so the way how I'm getting this error is when I'm trying to make an edit for a message in the ShutdownEvent for some type of thing that I'm working with, first, I thought maybe it gives me a limited request so I decided to start with one request only and try, and the error still coming to me. I tried to get any information for why would i get this breakpoint but there is no explanation for this type of error, idk if there is something wrong I'm doing could someone help me, please.

**Comment by Tais993** [comment, marked as answer]
The shutdown event triggers when JDA is off, you cannot run any new requests. (The requester has been stopped)

**Reply by OMAR-3OOV** [reply]
i mean yeah, but what is the point of putting this event then?

**Reply by Tais993** [reply]
So people know that the bot is offline, and they can do **non JDA** things.

**Reply by MinnDevelopment** [reply]
You can use [onStatusChange](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/events/StatusChangeEvent.html) and check for [SHUTTING_DOWN](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/JDA.Status.html#SHUTTING_DOWN) as an alternative.
